### PR TITLE
Add Geoffrey as an observer

### PR DESCRIPTION
**Pull request by GeoffreyBooth**
Opening a PR for this feels overly dramatic . . .

### CODE REVIEW

**Comment by jkrems** [comment]
Side note: One of the downsides of doing this via PR I just realized is that we make `ncu-team-sync` unhappy. Not sure if that's possible (or worth) addressing.

**Comment by jdalton** [comment]
> Opening a PR for this feels overly dramatic . . .

File: README.md [hunk 1]
```diff
@@ -62,7 +62,9 @@ Work includes:
 
 <!-- ncu-team-sync end -->
 
-<!-- ## Observers -->
+## Observers
+
+- [@GeoffreyBooth](https://github.com/GeoffreyBooth/) - Geoffrey Booth
```
Comment (MylesBorins) [inline README.md]: can you put this below  ` <!-- ncu-team-sync.team(nodejs/modules-observers) -->`. This is where the content would generally be autogenerated.

**Comment by MylesBorins** [comment]
@jkrems I'll make sure it is up to date in github.

File: README.md (hunk 1 above)
Comment (GeoffreyBooth) [reply README.md]: Done.

**Comment by GeoffreyBooth** [comment]
I appreciate all the approvals and +1s, but is the idea that this PR needs to get 22 approvals (51% of members) before it’s merged in?

I’d just like to be able to speak at meetings. Could I perhaps be invited to the next meeting via some way that I can speak, on the assumption that this will get approved at that meeting?

**Comment by MylesBorins** [comment]
@GeoffreyBooth please see https://github.com/nodejs/modules/issues/71, you are more than welcome to join and participate in the meeting. Thank you for your patience.

**Comment by MylesBorins** [comment]
Landed in README

**Pull request closed without merging**